Content.Server/Atmos/EntitySystems/AtmosphereSystem.cs

```diff
@@ -85,9 +85,15 @@ public override void Update(float frameTime)
 
             foreach (var (exposed, transform) in EntityManager.EntityQuery<AtmosExposedComponent, TransformComponent>())
             {
-                var tile = GetTileMixture(transform.Coordinates);
-                if (tile == null) continue;
-                var updateEvent = new AtmosExposedUpdateEvent(transform.Coordinates, tile);
+                // Used for things like disposals/cryo to change which air people are exposed to.
+                var airEvent = new AtmosExposedGetAirEvent();
+                RaiseLocalEvent(exposed.Owner, ref airEvent);
```

Comment:
Since the event itself does not contain any information about the EntityUid, this shouldn't be broadcast.

Reply:
```suggestion
                RaiseLocalEvent(exposed.Owner, ref airEvent, false);
```

Reply:
yep, oops